app/Console/Commands/UpdateSignup.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Signup;
+use Illuminate\Console\Command;
+use Rogue\Services\Three\SignupService;
+
+class UpdateSignup extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:updatesignups {--target= : The name of the field to update} {--targetValue= : The value to update the target with} {--campaign= : The campaign_id to search for signups under} {--campaign_run= : The campaign_run_id to search for signups under} {--date= : Will be used to search for signups greater than the provided value}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Update a target field on a set of signups contrained by the provided parameters';
+
+    /**
+     * The signup service instance.
+     *
+     * @var Rogue\Services\Three\SignupService
+     */
+    protected $signups;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(SignupService $signups)
+    {
+        parent::__construct();
+
+        $this->signups = $signups;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $targetField = $this->option('target') ?? null;
+        $targetValue = $this->option('targetValue') ?? null;
+
+        if (! $targetField) {
+            $this->error('No target field specified.');
+
+            return;
+        }
+
+        if (! $targetValue) {
+            $this->error('No target value specified.');
+
+            return;
+        }
```

Comment:
I like that you made these options even though they are required because it feels easier to keep track of what is what when you are writing out the command to run like this.